### DISCUSSION TITLE
Import - "Text codes"

### DISCUSSION THREAD

**Discussion by OtaStrom** [SubtitleEdit/subtitleedit#6018] [Ideas]
Hi. I'd like to suggest a new function that would allow importing all the text from another subtitle file with the same number of subtitles. In the same way like **Import - Time codes** works for time codes. So, It would be **"Import - Text codes"**. I would make it easy to update subtitles from another version where some corrections were made.

**Comment by OmrSi** [comment]
You can easily do this by:
1. Select all lines in the first subtitle.
2. Use normal copy `Control + C`
3. Go to the second file.
4. Right-click on the first line.
5. Use column -> paste from clipboard, there is also a shortcut for this.
6. Select `Text only` and Okay.

Might seem a bit complicated, but you get used to it, it's very easy and helpful.
